solution/ui/regulations/eregs-vite/src/components/JumpTo.vue

```diff
@@ -1,15 +1,13 @@
 <template>
     <div>
-        <div class="jump-to-label">Jump to Regulation Section</div>
         <form @submit.prevent="getLink">
             <div class="jump-to-input">
                 <select
                     v-if="defaultTitle === ''"
-                    id="jumpToTitle"
                     v-model="selectedTitle"
                     name="title"
                     aria-label="Regulation title number"
-                    class="ds-c-field"
+                    class="jump-to-title ds-c-field"
```

Comment:
Is it better to have this as a class than an ID?  On the cypress test we end up doing a .first() which to me in the sense of testing implies that the same elemenet can be used multiple times on the same page, which is not the case.  jump to should only be on the header.  If there is a .second(I'm either making this up or this is a thing) that would be bad.  The Id's allow us to with consistency apply values to those boxes.  I'm sure either way is fine and is just a philosopical difference but I could be wrong

Reply:
having it as an ID when there were two instances of the Jump To on the homepage failed the accessibility tests because, as you say, having an ID implies something can only be used once on a page.  But since there will be a brief moment in time where there are two Jump To components on the homepage, these were changed to classes.

These can be changed back to IDs as part of the Acceptance Criteria for the future story where we remove the Jump To component from the homepage hero banner.

Reply:
Although I'm talking through the homepage updates with Jonathan now and it might just make sense to remove the Jump To from the Hero banner in this story, which would allow me to revert these classes back to IDs and address your concerns here.

Reply:
I've changed these back to IDs and updated the tests.  Resolved!